src/System.Data.SqlClient/tests/ManualTests/SQL/SqlNamedPipesTest/SqlNamedPipesTest.cs

```diff
@@ -15,19 +15,13 @@ public static void ValidConnStringTest()
             SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(DataTestUtility.NpConnStr);
             builder.ConnectTimeout = 5;
 
-            string plainConnString = builder.ConnectionString;
-            builder.DataSource = "np:" + GetHostFromDataSource(builder.DataSource);
-            string serverNameOnlyConnString = builder.ConnectionString;
-
             if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                OpenBadConnection<PlatformNotSupportedException>(plainConnString);
-                OpenBadConnection<PlatformNotSupportedException>(serverNameOnlyConnString);
```

Comment:
Just having the server name in the connection string uses the default "sql/query" path, but not all SQL servers use a default path. Basic connectivity is already covered by just using the existing connection string.